### PR TITLE
arvo, landscape: fix upgrade flow

### PR DESCRIPTION
**Pull request by liam-fitzgerald**
- Fixes `|bump` json conversion
- Correctly identifies blocked desks, includes them in `except=`
- Fixes detection of a blocked %base update

### CODE REVIEW

**Review by arthyn — APPROVED** [review]

**Comment by liam-fitzgerald** [comment]
Screen Shot 2021-11-12 at 6 22 50 pm
cc: @urcades

File: pkg/grid/src/nav/notifications/SystemNotification.tsx [hunk 1]
```diff
@@ -7,9 +7,12 @@ import { Dialog, DialogClose, DialogContent, DialogTrigger } from '../../compone
 import { Elbow } from '../../components/icons/Elbow';
 import api from '../../state/api';
 import { useCharges } from '../../state/docket';
+import useKilnState, { useVat } from '../../state/kiln';
 
 import { NotificationButton } from './NotificationButton';
 import { disableDefault } from '../../state/util';
+import { Vat } from '../../../../npm/api/dist';
```
Comment (arthyn) [inline pkg/grid/src/nav/notifications/SystemNotification.tsx:14]:
```suggestion
import { Vat } from '@urbit/api';
```

File: pkg/grid/src/nav/Leap.tsx [hunk 2]
```diff
@@ -227,42 +227,49 @@ export const Leap = React.forwardRef(
       [selection, rawInput, match, matches, selectedMatch]
     );
 
+    console.log(menu);
```
Comment (arthyn) [inline pkg/grid/src/nav/Leap.tsx:230]:
```suggestion
```

**Review by arthyn — APPROVED** [review]
Few nits but seems good

**Pull request merged**